relayer/application_relayer.go

```diff
@@ -28,17 +28,11 @@ import (
 )
 
 const (
-	// Number of retries to collect signatures from validators
-	maxRelayerQueryAttempts = 5
-	// Maximum amount of time to spend waiting (in addition to network round trip time per attempt)
-	// during relayer signature query routine
-	signatureRequestRetryWaitPeriodMs = 10_000
+	retryMaxElapsedTime = 10 * time.Second
```

Comment:
I think we can rename the consts using `MaxElapsedTime` (ex: `retryMaxElapsedTime` -> `retryTimeout`) as we are now using `utils.DefaultRPCTimeout`. Otherwise we are mixing the use of `Timeout` and `MaxElapsedTime` when calling the function

Reply:
Sounds good to me! My main concern is that we have distinct `const`s for the various timeout scenarios.

Reply:
I have changed consts as `Timeout` and kept distinct ones 👍